package.json

```diff
@@ -29,7 +29,7 @@
 		"@emotion/styled": "^10.0.27",
 		"@guardian/ab-core": "7.0.1",
 		"@guardian/automat-modules": "^0.3.8",
-		"@guardian/commercial": "19.0.0",
+		"@guardian/commercial": "0.0.0-beta-20240603085241",
```

Comment:
FYI, I’m inclined not to approve this PR untilthis point to an non-canary version to prevent an inadvertent release.

Reply:
I understand. I am waiting until I get an approval from Commercial, merge and then bump the version in here. It is good to make this PR ready for an approval after addressing all the comments and merge Commercial.

Reply:
Done 420cd3bf2ec92e144cc0eafc014f5d4733da3e76